packages/neo-one-smart-contract-compiler/src/api.ts

```diff
@@ -20,7 +20,7 @@ export const compileContract = async (
 ): Promise<CompileContractResult> => {
   const result = await compileContractBase({ filePath, host, linked });
 
-  throwOnDiagnosticErrorOrWarning(result.diagnostics, ignoreWarnings);
+  throwOnDiagnosticErrorOrWarning(result.diagnostics, ignoreWarnings, 'Error while compiling contract(s)');
```

Comment:
Let's put the contract name here to make it a little cleaner. So it's `'Error while compiling contract ${contractName}'`

And change `_contractName` to `contractName`